ALU_wrapper.py

```diff
@@ -82,47 +63,36 @@ def sendtoslims():
 
 
 def compressfiles(folderpath):
-    destination = seqpath + "/saved/"  # This needs changing. Now set to subfolder saved under input-path (seqpath).
-    filename = folderpath.split('/').pop() + ".tgz"
+    destination = seqpath + "/saved/"  # This needs changing. Now set to sub-folder saved under input-path (seqpath).
+    filename = folderpath.split('/').pop() + ".tar.gz"
     subprocess.call(['tar', 'zcf', destination + filename, folderpath])
 
 
-if not args.run:
-    print("Not parsing any found files, specify -r to actually run")
-folderList = next(os.walk(seqpath + '/.'))[1]
-if args.verbose:
-    print("List of folders in dir: ")
-    for folder in folderList:
-        print(folder)
-    print("--- END OF LIST ---")
+def mailout():
+    msg_from = "crc@gu.se"  # TODO: Get from config-file
+    msg_subject = "The crc files are done!"
+    msg_body = "Hi!\nThe CLC-files are processed and ready for download."  # TODO: Change to a better mail
+    emailer.send_email(msg_to, msg_from, msg_subject, msg_body)
 
+
+folderList = next(os.walk(seqpath + '/.'))[1]
 for folder in folderList:
+    logging.info("Folder: " + folder)
     if "DNA" in folder and "_" not in folder:
-        # Skipping folders with _ in the names... this is actually not needed.
+        # Skipping folders with _ in the names. This is not actually needed.
         path = seqpath + '/' + folder
-        if glob.glob(path + '/*melt*.vcf*'):  # depending on how we want to run this. just an example.
-            if args.verbose:
-                print("Processed files found in folder " + folder + ", not running again.")
+        if glob.glob(path + '/*melt*.vcf*'):  # TODO: Get regex from config-file
+            logging.debug("Processed files found in folder " + folder + ", not running again.")
```

Comment:
f-string?
`logging.debug(f"Processed files found in folder {folder}, not running again.")`

Reply:
Done in 9a80084